app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -22,6 +22,16 @@
   <div class="govuk-grid-column-two-thirds">
   <%= render ProviderInterface::StatusBoxComponent.new(application_choice: @application_choice) %>
   </div>
+  <% unless rendering_pdf? %>
```

Comment:
I think including the `print.css` stylesheet could make this work without the need for logic in the template. I'm not sure the `pdfkit` middleware supports setting the `stylesheet` property on the instance of `PDFKit`.

What would you think about adding a `respond_to { |format| format.pdf etc }` in the show action in the `ApplicationChoicesController`, and manually instantiating a `PDFKit` there?

Reply:
We can set `print_media_type` in the config which will use the print version of styles - will update :+1:

Reply:
We could make a similar method to the middleware and call it in a `respond_to` block.
It seems like the middleware intercepts and rewrites `response.body` and a few headers so this could presumably be wrapped around `render`.

Reply:
@duncanjbrown updated to use print styles here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1572/commits/f657bb8614c9f9dbc198315c0ff1ab083889e61f
and template logic is removed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1572/commits/0a0c4c70a9b7538c6f7d181a050f9ab7e814c4a3